### PR TITLE
Natasha

### PR DESCRIPTION
**Pull request by natasha-sharma123**
General Updates

### CODE REVIEW

File: 10-spatial-estimation.Rmd [hunk 1]
```diff
@@ -1123,7 +1123,7 @@ leaflet() %>% addTiles() %>%
     title = "Predicted basal area")
 ```
 
-**Figure 10.20.** \ref{fig:figs} Predicted basal area over the entire Fort Saint Johns TSA using five nearest neighbor projected over the province of British Columbia.
+**Figure 10.20.** \ref{fig:figs} Predicted basal area over the entire Fort Saint Johns TSA using five nearest neighbor projected over the province of British Columbia. Data from City of Vancouver and licensed under the Open Government License - Vancouver.
```
Comment (ect123) [inline 10-spatial-estimation.Rmd:1126]: I don't think this is data from the City of Van @pauldpickell is there another statement that belongs here?

File: 09-raster-analysis-terrain-modelling.Rmd [hunk 2]
```diff
@@ -527,7 +527,7 @@ m2 = leaflet() %>%
 ```
 
 ```{r 9-leaflet-map-layers, echo=FALSE, out.width="75%", fig.cap=fig_cap}
-fig_cap <- paste("Input layers and output prediction raster for the Chowade Watershed Management basin. Animated figure can be viewed in the web browser version of the textbook. Melser, CC-BY-SA-4.0.")
+fig_cap <- paste("Input layers and output prediction raster for the Chowade Watershed Management basin. Animated figure can be viewed in the web browser version of the textbook.Data from City of Vancouver and licensed under the Open Government License - Vancouver. Melser, CC-BY-SA-4.0.")
```
Comment (ect123) [inline 09-raster-analysis-terrain-modelling.Rmd:530]: This should be omitted or changed to the appropriate attribution

File: 03-types-of-data.Rmd [hunk 3]
```diff
@@ -273,7 +273,7 @@ Methods Discussed:
 <iframe width=100% height=800px src='images/03-PoliceViolenceIncidents_BC.html' frameBorder="0" >
   <p>Your browser does not support iframes</p>
 </iframe>
-
+Data from City of Vancouver and licensed under the Open Government License - Vancouver
```
Comment (ect123) [inline 03-types-of-data.Rmd:276]: This statement should be omitted

File: 15-LiDAR-acquisition-and-analysis.Rmd [hunk 4]
```diff
@@ -341,7 +341,7 @@ m = leaflet() %>%
 
 ```{r 15-map, echo=FALSE, fig.cap = fig_cap}
 
-fig_cap <- paste("Explore different area-based metrics produced in the case study above. Hover over tile icon (top left) to turn different layers on or off. Animated figure can be viewed in the web browser version of the textbook. (Du Toit, CC-BY-4.0). NOTE: this map contains interactive content and is available online.")
+fig_cap <- paste("Explore different area-based metrics produced in the case study above. Hover over tile icon (top left) to turn different layers on or off. Animated figure can be viewed in the web browser version of the textbook. Data from City of Vancouver and licensed under the Open Government License - Vancouver (Du Toit, CC-BY-4.0). NOTE: this map contains interactive content and is available online.")
```
Comment (ect123) [inline 15-LiDAR-acquisition-and-analysis.Rmd:344]: Need to verify this is from the COV

**Comment by ect123** [comment]
Many of the images have been reformatted from .jpg to .png, but the paths to the images have not been changed in the Rmd files, so the images will be broken.

File: 03-types-of-data.Rmd (hunk 3 above)
Comment (pauldpickell) [reply 03-types-of-data.Rmd:276]: June indicated that this police violence web map will be removed from the chapter, so this is moot.

File: 15-LiDAR-acquisition-and-analysis.Rmd (hunk 4 above)
Comment (pauldpickell) [reply 15-LiDAR-acquisition-and-analysis.Rmd:344]: This attribution is correct.

**Review by pauldpickell — CHANGES_REQUESTED** [review]
@natasha-sharma123 Images in Chapter need to be renamed within the rmd and converted to png. All of these images should have the prefix `17-` and these also need to be updated in the rmd.

**Review by pauldpickell — CHANGES_REQUESTED** [review]
@natasha-sharma123 "Animated figure can be viewed in the web browser version of the textbook" statement is not needed for png files, only gifs, leaflets, and other interactive content.

**Review by pauldpickell — CHANGES_REQUESTED** [review]
@natasha-sharma123 Need to hyperlink the license text to the license webpage. For example, [Open Government License - Vancouver](https://opendata.vancouver.ca/pages/licence/)

**Comment by ect123** [comment]
Merged and deleted branch, with a few unresolved issues. They will be resolved in subsequent pull requests from branches specific to those issues.

**Pull request merged**